src/vos/tests/evt_ctl.c

```diff
@@ -69,6 +69,10 @@ static int			 ts_feats = EVT_FEAT_DEFAULT;
 
 #define ORDER_DEF_INTERNAL	13
 #define ORDER_DEF		16
+#define	ORDER_TEST_SIZE		200
+
+#define EVT_MIN_ORDER	4
```

Comment:
I see we don't include evt_priv.h.   I think it would be better even to just move these to src/include/evtree.h so we don't duplcate the settings in two places.   That would be self documenting the API as well.

Please remove these and move the definitions from evtree.c to the evtree.h header file and add a short doc line
e.g. /** Minimum tree order for an evtree */

and maybe add a blurb to evt_create that the order must be >= EVT_MIN_ORDER and <= EVT_MAX_ORDER.

Reply:
Done. Thanks.